src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -109,6 +114,11 @@ object SparkCassandraStress {
           ("spark.cassandra.input.split.size" -> arg.toString))
       } text {"Read input size"}
 
+      opt[Int]('u',"throughput") optional() action { (arg,config) =>
+        config.copy(sparkOps = config.sparkOps +
+          ("spark.cassandra.output.throughput_mb_per_sec" -> arg.toString))
```

Comment:
Lets drop this and all the other spark.cassandra.* settings  since we can set them with --conf anyway (if run.sh is setup correctly, this will let us drop a bunch of params)